code-challenges.test.js

```diff
@@ -26,26 +31,80 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+// describe('divisibleByThree', () => {
+//     it('that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.', () => {
+//         const object1 = { number: 15 }
+//         // Expected output: "15 is divisible by three"
+//         const object2 = { number: 0 }
+//         // Expected output: "0 is divisible by three"
+//         const object3 = { number: -7 }
+//         // Expected output: "-7 is not divisible by three"
+//         expect(divisibleByThree(object1)).toEqual("15 is divisible by three")
+//         expect(divisibleByThree(object2)).toEqual("0 is divisible by three")
+//         expect(divisibleByThree(object3)).toEqual("-7 is not divisible by three")
+//     })
+// })
+    //   ReferenceError: divisibleByThree is not defined
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// input: 3 individual number objects
+// output: return a message depicting whether or not the number inputted is divisible by 3
+
+// const divisibleByThree = (number) => {
+//     // use number as placeholder for the arguement that will be console logged
+//     if (number % 3 === 0 ){
+//         // use mod % op to check if number is divis by three
+//         return `${number} is divisible by three`
+//         // if true, return message above
+//     } else {
+//         return `${number} is not divisible by three`
+//         // if false, or all else, return message above
+//     }
+// }
```

Comment:
Remember that the function is taking an object as an argument. That being said, you can't perform math on an object, so there will be an error when the code tries to do % 3 on an object. You first need to access the value in the object using dot notation.